### PR TITLE
Dummy PR

### PR DESCRIPTION
**Pull request by lemeurherveCB**
Trying to reproduce the `atest` strange linter error cf https://matrix.to/#/!TQpOBNqKouILtnYnAd:gitter.im/$c9R6t681xgJxxLJk1UQ3aBBI4RPrwyUc0WMyagQdpao?via=gitter.im&via=matrix.org&via=matrix.freyachat.eu

Fix #XXX

## Test

To test this pull request, you can run the following commands:

```shell
cd <to_package_directory>
go test
```

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -11,6 +11,8 @@ link:https://img.shields.io/github/actions/workflow/status/updatecli/updatecli/g
 link:https://api.securityscorecards.dev/projects/github.com/updatecli/updatecli[image:https://api.securityscorecards.dev/projects/github.com/updatecli/updatecli/badge[OpenSSF Scorecard]]
 link:https://bestpractices.coreinfrastructure.org/projects/6731[image:https://bestpractices.coreinfrastructure.org/projects/6731/badge[OpenSSF Best Practices]]
 
+Intentional atest
```
Comment (lemeurherveCB) [inline README.adoc:14]:
```suggestion
Intentional whaterverLatest
```

File: README.adoc [hunk 2]
```diff
@@ -11,6 +11,8 @@ link:https://img.shields.io/github/actions/workflow/status/updatecli/updatecli/g
 link:https://api.securityscorecards.dev/projects/github.com/updatecli/updatecli[image:https://api.securityscorecards.dev/projects/github.com/updatecli/updatecli/badge[OpenSSF Scorecard]]
 link:https://bestpractices.coreinfrastructure.org/projects/6731[image:https://bestpractices.coreinfrastructure.org/projects/6731/badge[OpenSSF Best Practices]]
 
+Intentional whaterverLatest
```
Comment (lemeurherveCB) [inline README.adoc:14]:
```suggestion
Intentional thingLatest
```

File: README.adoc [hunk 3]
```diff
@@ -11,6 +11,8 @@ link:https://img.shields.io/github/actions/workflow/status/updatecli/updatecli/g
 link:https://api.securityscorecards.dev/projects/github.com/updatecli/updatecli[image:https://api.securityscorecards.dev/projects/github.com/updatecli/updatecli/badge[OpenSSF Scorecard]]
 link:https://bestpractices.coreinfrastructure.org/projects/6731[image:https://bestpractices.coreinfrastructure.org/projects/6731/badge[OpenSSF Best Practices]]
 
+Intentional thingLatest
```
Comment (lemeurherveCB) [inline README.adoc:14]:
```suggestion
Intentional "-latest",
```

File: README.adoc [hunk 4]
```diff
@@ -11,6 +11,8 @@ link:https://img.shields.io/github/actions/workflow/status/updatecli/updatecli/g
 link:https://api.securityscorecards.dev/projects/github.com/updatecli/updatecli[image:https://api.securityscorecards.dev/projects/github.com/updatecli/updatecli/badge[OpenSSF Scorecard]]
 link:https://bestpractices.coreinfrastructure.org/projects/6731[image:https://bestpractices.coreinfrastructure.org/projects/6731/badge[OpenSSF Best Practices]]
 
+Intentional "-latest",
```
Comment (lemeurherveCB) [inline README.adoc:14]:
```suggestion
Intentional -latest
```

**Comment by jsoref** [comment]
It's coming from: https://github.com/updatecli/updatecli/blob/d71451b35ef47a7e89b200f404538c5d544aa35b/.github/actions/spelling/patterns.txt#L53

You'll want to adjust that pattern (the simplest thing is to remove the `l`)
```diff
-(?:^|[\t ,>"'`=(])-[DPWXYLlf](?=[A-Z]{2,}|[A-Z][a-z]|[a-z]{2,}) 
+(?:^|[\t ,>"'`=(])-[DPWXYLf](?=[A-Z]{2,}|[A-Z][a-z]|[a-z]{2,}) 
```

It's possible for check-spelling to identify which patterns produced a fractional term, but it's somewhat expensive (it'd involve running all the patterns against the line until it identifies just the ones that contribute).

I'm not sure if `@prerelease` would improve this case (I'm currently cleaning up all of the commits I have in prerelease... hoping to have a clean test to release really soon).

**Comment by lemeurherveCB** [comment]
Opened https://github.com/updatecli/updatecli/issues/2804, thanks again @jsoref!

**Pull request closed without merging**